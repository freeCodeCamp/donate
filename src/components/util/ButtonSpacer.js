import React from 'react';

import './button-spacer.css';

function ButtonSpacer() {
  return <div className='button-spacer' />;
}

ButtonSpacer.displayName = 'ButtonSpacer';

export default ButtonSpacer;
