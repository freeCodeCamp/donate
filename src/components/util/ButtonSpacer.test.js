/* global expect */

import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ButtonSpacer from './ButtonSpacer';

Enzyme.configure({ adapter: new Adapter() });

test('<ButtonSpacer /> snapshot', () => {
  const component = renderer.create(<ButtonSpacer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
