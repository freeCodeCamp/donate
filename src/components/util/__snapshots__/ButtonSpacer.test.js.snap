// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonSpacer /> snapshot 1`] = `
<div
  className="button-spacer"
/>
`;
